torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -2275,8 +2275,9 @@ def call_dist_op(
                         work.wait()
 
             if expect_event and dist.get_backend() in PROFILING_SUPPORTED_BACKENDS:
+                # We are only interested in the backend's implementation not the dispatcher wrapper.
                 events = get_profiling_event(
-                    profiling_title_postfix, autograd_profiler_ctx
+                    dist.get_backend()+profiling_title_postfix, autograd_profiler_ctx
```

Comment:
Are these changes needed?

Reply:
If we don't add the backend prefix, the events returned will be the nccl::broadcast and newly added broadcast op. I think the intention here is to only capture the nccl::broadcast.